README.md

```diff
@@ -12,24 +12,27 @@ Bitcoin stock price for your RPi Zero
     ```
     Interfacing Options -> SPI
    ```
-2. Install eInk display drivers
+2. Installing dependencies via apt
     ```
-    git clone https://github.com/waveshare/e-Paper.git
-    cd e-Paper-master/RaspberryPi_JetsonNano/python/
-    pip3 install -e $(pwd)
+    sudo apt update
+    sudo apt install -y libtiff5 libopenjp2-7 libopenjp2-7 libatlas-base-dev
     ```
-    for more information refer to: https://www.waveshare.com/wiki/2.13inch_e-Paper_HAT
-3. Download Zero BTC Screen
+3. Install eInk display drivers
+    ```
+    pip3 install image numpy RPi.GPIO spidev Pillow
```

Comment:
`Image` is a bit of an overkill I'm afraid, it helped because it includes the `Pillow` library. However, it's already covered by the discontinued `python3-pil` as per the waveshares wiki.
Therefore, we can just add the following lines to save people time reading the waweshare's wiki:
```
sudo apt-get install python3-pip python3-pil python3-numpy
pip3 install RPi.GPIO spidev
```